### ISSUE TITLE
no page load time breakdown in ES??

### ISSUE THREAD

**Issue by gggopi** [stagemonitor/stagemonitor-mailinglist#50]
Hi,
I noticed that the page load time breakdown is not going to ES anymore with the new upgrade.  
Has it been removed? Why? Will it be back ? When?

**Comment by trampi** [comment]
Hi, the page load time breakdown is being replaced by a new feature, called EUM (end user monitoring).
I am writing the documentation for the new end user monitoring at this moment. It gives you a better insight into the different aspects of the timing of a browser. EUM is more fine grained and allows a better understanding of the different parts of a page load.

The current version of the documentation is in the link below, expect an update regarding the usage of this feature later today:
https://github.com/stagemonitor/stagemonitor/wiki/End-User-Monitoring

I am here if you have any further questions.

**Comment by felixbarny** [comment]
From 0.80.0 on, there is no page load time breakdown in the `stagemonitor-metrics*` index. Instead, there are `pageload` spans in the `stagemonitor-spans*` index. There will be a dashboard to have a more fine grained breakdown based on `pageload` spans.

**Comment by gggopi** [comment]
Thanks @trampi  @felixbarny 

Does this have separate a dashboard? Name?

**Comment by felixbarny** [comment]
The dashboard is not yet finished but we have prototyped a visualization. You can import it via Kibana > Management > Saved Objects (unzip the json first)

[PageLoadTimeBreakdown.json.zip](https://github.com/stagemonitor/stagemonitor-mailinglist/files/1162277/PageLoadTimeBreakdown.json.zip)

But you will have to enable end user monitoring first: https://github.com/stagemonitor/stagemonitor/wiki/End-User-Monitoring#option-1-automatic-end-user-monitoring-via-automatic-injection

**Comment by gggopi** [comment]
The js loaded in the browser is giving a 204 response code.  I dont see the pageload metrics in ES. 
Checked http://es_ip:port/stagemonitor-spans-[YYYY.MM.DD]/pageload/_search

**Comment by trampi** [comment]
Any log messages in the monitored application? Like `limit of total fields [1000] in index`?

**Comment by felixbarny** [comment]
The index type is not `pageload`. Try `/stagemonitor-spans*/_search?q=type:pageload`

**Comment by gggopi** [comment]
Yes. Got it. Metrics are there. Imported viz works fine. Thanks. @felixbarny @trampi
